examples/compiled/trellis_bar_histogram_label_rotated.vg.json

```diff
@@ -135,6 +135,12 @@
                     "update": {
                         "angle": {
                             "value": -90
+                        },
+                        "align": {
+                            "value": "right"
```

Comment:
This is bad ... if you look at the SVG diff - the old SVG is clearly better.  So please revise your rule.

Reply:
Forgot to rebuild..